### PR TITLE
New functionality and fixes for trips

### PR DESCRIPTION
**Pull request by jamescw**

### CODE REVIEW

File: EquiTrack/assets/js/trips_admin.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        });
+
+        $("div[id*='-arrive']").datepicker({
+            format: "yyyy-mm-dd",
+            todayBtn: "linked",
+            autoclose: true,
+            todayHighlight: true
+        });
+
+    });
```
Comment (jamescw) [inline EquiTrack/assets/js/trips_admin.js]: Did this work in the end?

File: EquiTrack/trips/forms.py [hunk 2]
```diff
@@ -29,17 +31,20 @@ def clean(self):
...
+                depart = depart.date()
+                arrive = arrive.date()
+
+                if to_date < depart or depart < from_date or to_date < arrive or arrive < from_date:
+                    raise ValidationError(
+                        'Travel dates must be within overall trip dates'
+                    )
+
+        if self.has_changed() and self.instance.trip.status == Trip.APPROVED:
+            self.instance.trip.status = Trip.PLANNED
```
Comment (jamescw) [inline EquiTrack/trips/forms.py]: This works?

File: EquiTrack/assets/js/trips_admin.js (hunk 1 above)
Comment (ntrncic) [inline EquiTrack/assets/js/trips_admin.js]: no it didn't

File: EquiTrack/trips/models.py [hunk 3]
```diff
@@ -340,7 +345,7 @@ class ActionPoint(models.Model):
     actions_taken = models.TextField(blank=True, null=True)
     completed_date = models.DateField(blank=True, null=True)
     comments = models.TextField(blank=True, null=True)
-    closed = models.BooleanField(default=False)
+    closed_choice = models.CharField(choices=CLOSED, max_length=254, null=True, verbose_name='Status')
```
Comment (jamescw) [inline EquiTrack/trips/models.py]: can we call this 'status' instead (sorry!)

**Comment by ntrncic** [comment]
I think we are good to merge this now @jamescw

**Pull request merged**